.travis.yml

```diff
@@ -3,6 +3,7 @@ node_js:
 - '8'
 cache: yarn
 script:
+- yarn lint 
```

Comment:
Everything looks good in this PR. 

Was curious why this step has been added to the pipeline? Is it necessary considering that we all have to run it to commit?

Reply:
Yes. If the lint config changes then lint errors can slip through, since lint-staged only runs against staged files.